src/tooltip/index.tsx

```diff
@@ -1,16 +1,53 @@
 import type {
+    ButtonProps as RACButtonProps,
     TooltipProps as RACTooltipProps,
     TooltipTriggerComponentProps as RACTooltipTriggerComponentProps,
 } from "react-aria-components";
 
 import clsx from "clsx";
+import { forwardRef } from "react";
 import {
+    Button as RACButton,
     OverlayArrow as RACOverlayArrow,
     Tooltip as RACTooltip,
     TooltipTrigger as RACTooltipTrigger,
 } from "react-aria-components";
 
-import { overlayArrowCSS, overlayArrowSvgCSS, tooltipCSS } from "./styles.css";
+import {
+    overlayArrowCSS,
+    overlayArrowSvgCSS,
+    tooltipCSS,
+    tooltipFieldButtonCSS,
+} from "./styles.css";
+
+/** ---------------------------------------------
+ * Trigger Button
+ * ----------------------------------------------- */
+
+export type TriggerButtonProps = RACButtonProps;
+
+/**
+ * A field button is a button that is intended to be used inside a `Group` component
+ * to add additional functionality to a field. The `slot` prop is used to connect the
+ * button to the field.
+ */
+export const TooltipFieldButton = forwardRef<
```

Comment:
I feel the naming is off — maybe `TooltipTriggerButton` instead?

Reply:
Sweet - that's what I had it as originally.